src/renderer/store/modules/utils.js

```diff
@@ -315,7 +315,7 @@ const actions = {
     let urlType = 'unknown'
 
     const channelPattern =
-      /^\/(?:(?:channel|user|c)\/)?(?<channelId>[^/]+)(?:\/(?<tab>join|featured|videos|playlists|about|community|channels))?\/?$/
+      /^\/(?:(?:channel|user|c)\/)?(?<channelId>[^/]+)(?:\/(?<tab>join|featured|videos|live|playlists|about|community|channels))?\/?$/
```

Comment:
Invidious uses `streams` instead of `live` so we should probably handle that too

Reply:
Added it.